lib/modding/IdentifierStorage.cpp

```diff
@@ -126,7 +126,7 @@ CIdentifierStorage::ObjectCallback CIdentifierStorage::ObjectCallback::fromNameW
 
 CIdentifierStorage::ObjectCallback CIdentifierStorage::ObjectCallback::fromNameAndType(const std::string & scope, const std::string & type, const std::string & fullName, const std::function<void(si32)> & callback, bool optional, bool bypassDependenciesCheck, bool caseSensitive)
 {
-	assert(!scope.empty());
+	//assert(!scope.empty());
```

Comment:
Leftover from debug? This generally means corrupted request

Same for another edit in this file